File: 240206/007/007.py
class Place:
    def __init__(self,secret_code,meeting_point,time):
        self.secret_code = secret_code
        self.meeting_point = meeting_point
        self.time = time

place1 = Place('codetree','L',13)

print("secret code : ",place1.secret_code)
print("meeting point : ",place1.meeting_point)
print("time : ",place1.time)

File: 250119/007/007.java
import java.util.*;

class Answer {
    String secretCode;
    char meetingPoint;
    int time;

    public Answer(String secretCode, char meetingPoint, int time) {
        this.secretCode = secretCode;
        this.meetingPoint = meetingPoint;
        this.time = time;
    }
};


public class Main {

    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);

        String sCode;
        char mPoint;
        int time;

        sCode = sc.next();
        mPoint = sc.next().charAt(0);
        time = sc.nextInt();

        Answer answer = new Answer(sCode, mPoint, time);

        System.out.println("secret code : " + answer.secretCode);
        System.out.println("meeting point : " + answer.meetingPoint);
        System.out.println("time : " + answer.time);

    }
}